service/api_logic/sports_logic.py

```diff
@@ -1,17 +1,35 @@
 from database.models import Sport
-from database.session import SessionLocal
 from exept.colors_text import print_error_message
+#from database.models.league import League
 
-session = SessionLocal()
+def get_all_sports(session):
+    try:
+        sports = session.query(Sport).all()
+        if not sports:
+            print_error_message("No sports found in the database")
+            return [], 204
+        return [
+            {
+                "sport_id": sport.sport_id,
+                "sport_name": sport.sport_name,
+                "sport_img": sport.sport_img
+            } for sport in sports], 200
+    except Exception as e:
+        raise Exception
 
-def get_all_sports():
-    sports = session.query(Sport).all()
-    if not sports:
-        print_error_message("No sports found in the database")
-        return [], 204
-    return [
-        {
-            "sport_id": sport.sport_id,
-            "sport_name": sport.sport_name,
-            "sport_img": sport.sport_img
-        } for sport in sports], 200
\ No newline at end of file
+'''
```

Comment:
commented code


BTW, I'm pretty sure that is not the proper way of commenting in python. You should use '#' on each new line. You can do that by using CTRL+/ 9by default in most code editors) after selecting the code you're willing to comment